src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1632,13 +1632,19 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
 
         #region Various HTTP Method Tests
 
-        [ActiveIssue(22161, TargetFrameworkMonikers.Uap)]
         [OuterLoop] // TODO: Issue #11345
         [Theory, MemberData(nameof(HttpMethods))]
         public async Task SendAsync_SendRequestUsingMethodToEchoServerWithNoContent_MethodCorrectlySent(
             string method,
             bool secureServer)
         {
+            if (PlatformDetection.IsUap && method == "TRACE")
+            {
+                // 'TRACE' method is not supported on Uap.
+                // See: https://github.com/dotnet/corefx/issues/22161
+                return;
```

Comment:
It already throws an HttpRequestException since we get an error back from the WinRT layer:

>System.Net.Http.HttpRequestException : An error occurred while sending the request.
---- System.Runtime.InteropServices.COMException : The text associated with this error code could not be found.
The requested operation is invalid

We plan to leave that in case someday, Wininet changes their mind and allows the method again.  There is no real added value for developer to see the PNSE.  And we have already documented this on the AppCompat wiki.